specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2023-08-01/dataprotection.json

```diff
@@ -0,0 +1,7636 @@
+          "description": "Gets or sets the exclude namespaces property. This property sets the namespaces to be excluded during restore from vault.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "includedResourceTypes": {
+          "description": "Gets or sets the include resource types property. This property sets the resource types to be included during restore from vault.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "excludedResourceTypes": {
+          "description": "Gets or sets the exclude resource types property. This property sets the resource types to be excluded during restore from vault.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "labelSelectors": {
+          "description": "Gets or sets the LabelSelectors property. This property sets the resource with such label selectors to be included during restore from vault.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "persistentVolumeRestoreMode": {
+          "description": "Gets or sets the PV (Persistent Volume) Restore Mode property. This property sets whether volumes needs to be restored from vault.",
+          "enum": [
+            "RestoreWithVolumeData",
+            "RestoreWithoutVolumeData"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PersistentVolumeRestoreMode",
+            "modelAsString": true
+          }
+        },
+        "conflictPolicy": {
+          "description": "Gets or sets the Conflict Policy property. This property sets policy during conflict of resources during restore from vault.",
+          "enum": [
+            "Skip",
+            "Patch"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ExistingResourcePolicy",
+            "modelAsString": true
+          }
+        },
+        "namespaceMappings": {
+          "description": "Gets or sets the Namespace Mappings property. This property sets if namespace needs to be change during restore from vault.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "restoreHookReferences": {
+          "description": "Gets or sets the restore hook references. This property sets the hook reference to be executed during restore from vault.",
+          "items": {
+            "$ref": "#/definitions/NamespacedNameResource"
+          },
+          "x-ms-identifiers": [],
+          "type": "array"
+        },
+        "stagingResourceGroupId": {
+          "description": "Gets or sets the staging RG Id for creating staging disks and snapshots during restore from vault.",
+          "type": "string"
+        },
+        "stagingStorageAccountId": {
```

Comment:
Is it an azure resource ID? If so, please add "format": "arm-id" and "x-ms-arm-id-details".

https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details